### DISCUSSION TITLE
Search tabs

### DISCUSSION THREAD

**Discussion by chriswu000** [brookhong/Surfingkeys#1404] [Ideas]
Hey, this extension is absolutely amazing!

I had a question, and I might have missed this in the documentation, but is there a way to search across tabs?  Perhaps as you search, it filters a list of open tabs against the title and metadata, (with favicons for easier identification)?  Then on select, it takes you to that tab?

A little bit like this extension (Session Buddy), but in a "vim" way.  Treating the tabs like buffers that you can list and search.  Thanks!
https://chrome.google.com/webstore/detail/session-buddy/edacconmaakjimmfgnblocblbcdcpbko?hl=en

**Comment by jo-so** [comment]
Is the key `T` what you are looking for?

**Reply by ben-wall** [reply]
No I just tried that and I think he means searching the text on the page across tabs.
